content/source/docs/enterprise-beta/api/modules.html.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: enterprise2
+page_title: "Modules - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-modules"
+---
+
+# Registry Modules API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## Listing and reading modules, providers and versions
+
+The Terraform Enterprise Module Registry provides the same API endpoints as the [Terraform Registry HTTP API](../../registry/api.html). Terraform Enterprise extends this API to add [authentication](./index.html#authentication). Refer to the [Module Registry HTTP API](../../registry/api.html) to perform the following:
+
+- List available versions for a specific module
```

Comment:
I have a PR ready to go to add listing modules and search modules in the public API docs. I'll post back here if this is still open or if not I'll try make sure this is updated once that lands.